dbms/src/Storages/DeltaMerge/tests/gtest_dm_segment.cpp

```diff
@@ -421,27 +420,21 @@ try
 }
 CATCH
 
-class SegmentDeletionRelevantPlace_test
-    : public Segment_test
+class SegmentDeletionRelevantPlaceTest
+    : public SegmentTest
     , public testing::WithParamInterface<bool>
 {
-    DB::Settings getSettings()
-    {
-        DB::Settings settings;
-        auto enable_relevant_place = GetParam();
-
-        if (enable_relevant_place)
-            settings.set("dt_enable_relevant_place", "1");
-        else
-            settings.set("dt_enable_relevant_place", "0");
-        return settings;
-    }
```

Comment:
This is a useless function that never called after https://github.com/pingcap/tiflash/pull/2415